src/Chainweb/Version.hs

```diff
@@ -623,21 +623,18 @@ coinV2Upgrade Mainnet01 cid h
     | cid == unsafeChainId 8 = h == 140808
     | cid == unsafeChainId 9 = h == 140808
     | otherwise = error $ "invalid chain id " <> sshow cid
-coinV2Upgrade Development cid h
-    | cid == unsafeChainId 0 = h == 3
-    | otherwise = h == 4
```

Comment:
This breaks two test cases in `test/Chainweb/Test/Pact/TransactionTests.hs`. Those test should probably be ported to use chainweb version `Mainnet01`.